packages/wallets/src/features/cashier/modules/DepositFiat/DepositFiat.tsx

```diff
@@ -17,7 +18,7 @@ const DepositFiat = () => {
         }
     }, [isAuthorizeSuccess, mutate]);
 
-    if (isError) return <p>Error</p>;
+    if (isError) return <WalletsErrorScreen message={(error as Error).message} />;
```

Comment:
@lubega-deriv 
don't think that this Error type is what we need. 
let leave an implementation with `isServerError`:
if (isError && isServerError(error)) return <WalletsErrorScreen message={error.message} />;

here we are really checking our error that this is a server error, and not relying on our assertion.
Using `as` should be a last resort. IMHO that means that smth wrong with our type system

Reply:
I agree with @heorhi-deriv. Lets avoid using as